### ISSUE TITLE
Übersetzung/Label für Grid-Klassen ermöglichen

### ISSUE THREAD

**Issue by denniserdmann** [ErdmannFreunde/contao-grid-bundle#8]
Immer wieder kommt der Wunsch auf, dass Klassen wie col-md-4 in _für nichtentwickler verständliche Sprache_ übersetzt werden könnte.

Wäre es ohne BC möglich, die Klassen zusätzlich mit „Übersetzungen“ auszustatten?

**Comment by richardhj** [comment]
Technisch ist es sehr einfach, weil wir nur das DCA erweitern müssen:

```php
$GLOBALS['TL_DCA']['tl_content']['fields']['grid_columns'] = [
    'options_callback' => [GridColsOptionsListener::class, 'onOptionsCallback'],
    'reference' => [
        'col-md' => 'Medium Screen',
        'col-md-5' => '5/12 Spalte (Medium Screen)'
    ],
];
```

Konzeptionell wird es schwieriger, weil wir nicht immer von der Standard-Konfiguraiton mit 12 Spalten ausgehen können, da die CSS-Klassen konfigurierbar sind. 
Können wir mit dem Bundle Übersetzungen mitliefern?

Ausserdem würden wir die human-readable CSS-Klassen sicherlich als Opt-In bereitstellen, konfigurierbar über die config.yml, und hier ist die Frage, ob das nicht wieder zu technsich für Nicht-Entwickler ist.

**Comment by denniserdmann** [comment]
Ja, der Gedanke war, das über die config.yml als Opt-In zu lösen. Da kann der Entwickler dann auch auf die abweichende Spalten-Anzahl eingehen und seine eigenen Übersetzungen schreiben. 

Eine andere Idee war, ob man die Grid-Klassen über die **symfony/twig/contao translations** auch im Backend übersetzen könnte. Dann könnte jeder Entwickler sich seine Übersetzungen über eine grid.de.yml zusammenstellen. Geht das?

Vielleicht so?

```
$GLOBALS['TL_DCA']['tl_content']['fields']['grid_columns'] = [
    'options_callback' => [GridColsOptionsListener::class, 'onOptionsCallback'],
    'reference' => [
        'col-md'     => $this->translator->trans('GRID.col-md', [], 'contao_default'),
        'col-md-5'   => $this->translator->trans('GRID.col-md-5', [], 'contao_default')
    ],
];
```

Besser wäre natürlich automatisiert.

**Comment by richardhj** [comment]
Habe die ersten Übersetzungen in a173a13a561fc000e15af69973d287f0d84ed6ad angelegt.

Opt-In:
```yml
erdmannfreunde_contao_grid:
  translated_labels: true
```

Überschreiben der Übersetzungen: einfach über /contao/languages/xx/default.php
